R/test-example.R

```diff
@@ -4,13 +4,26 @@
 #' Each example counts as one test, and it succeeds if the code runs without
 #' an error.
 #'
-#' @param path For \code{test_examples}, path to directory containing Rd files.
+#' @param path For \code{test_examples}, path to the package directory.
 #'   For \code{test_example}, path to a single Rd file. Remember the working
 #'   directory for tests is \code{tests/testthat}.
+#' @param rd A parsed Rd object, obtained from \code{\link[tools]{Rd_db}}.
 #' @export
-test_examples <- function(path = "../../man") {
-  man <- dir(path, "\\.Rd$", full.names = TRUE)
-  lapply(man, test_example)
+test_examples <- function(path = "../..") {
```

Comment:
I _think_ this would be clearer with two functions - maybe `test_examples_installed` and `test_examples_source` ?